### DISCUSSION TITLE
Filter Starred Repositories

### DISCUSSION THREAD

**Discussion by ProgrammingParadox** [community/community#41582] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

While trying to organize my stars, I started struggling to find repositories I hadn't starred yet. To improve such scenarios, perhaps allow the user to filter by "not in a list", or by certain list groups (eg. I want to see only my repositories that are in the groups "A" or "B" but not "C")

**Comment by radar-blip** [comment]
Your idea has been suggested a couple times before but not yet implemented. 
Maybe sometime in the near future we will have this option. 👍🏻

**Reply by 66667888sssttt** [reply]
- [x] #

**Reply by Svengd** [reply]
> - [x] #

**Reply by Kobra481** [reply]
> - [x] #

**Comment by mahmudul0100** [comment]
Hi I am Mahmudul

**Comment by mahmudul0100** [comment]
Apni ke

**Comment by Hnuri77** [comment]
Find I phone

**Comment by mahmudul0100** [comment]
I am hacker

**Comment by 66667888sssttt** [comment]
[]()

**Reply by 66667888sssttt** [reply]
###- I'm

**Comment by 66667888sssttt** [comment]
> []()

Blue

**Comment by mahmudul0100** [comment]
Cool me 01617841630

**Comment by TalleA15** [comment]
I'm saidu idi talle
a  Computer Cyber Security

**Reply by mahmudul0100** [reply]
I si

**Reply by TalleA15** [reply]
@ programming to stop Cyber Crime 
- Like hackers of  lmforation , - [ ] _[]()~~@**** ~~_

**Reply by mahmudul0100** [reply]
Sar I hack Facebook account so you are help me

**Reply by TalleA15** [reply]
- [ ] - Student @ NITDA @ blockchain[]- [ ] - [ ] # - ()~~~~****~~~~

**Comment by mahmudul0100** [comment]
Sorry I no

**Comment by AQAqoma** [comment]
- [ ] # ~~~~

**Comment by AQAqoma** [comment]
- [ ] # ~~~~

**Comment by AQAqoma** [comment]
> hadn't
